atomic_reactor/plugins/koji_import.py

```diff
@@ -532,12 +533,11 @@ def run(self):
         if build_id is not None and build_token is not None:
             koji_metadata['build']['build_id'] = build_id
 
+        self.upload_metadata(koji_metadata, server_dir)
         try:
-            if build_token:
```

Comment:
is it safe to remove this if ?

Reply:
If not present, it will have value None. and that is the default value for token in CGImport method anyway.

Reply:
So I would say yes.